### ISSUE TITLE
groundhog-th-0.10 build failure

### ISSUE THREAD

**Issue by DanBurton** [lykahb/groundhog#84]
As seen on the Stackage build server:

```
[3 of 3] Compiling Database.Groundhog.TH ( Database/Groundhog/TH.hs, dist/build/Database/Groundhog/TH.o )

Database/Groundhog/TH.hs:52:1: error:
    Could not load module ‘Text.Libyaml’
    It is a member of the hidden package ‘libyaml-0.1.0.0’.
    Perhaps you need to add ‘libyaml’ to the build-depends in your .cabal file.
    Use -v to see a list of the files searched for.
   |
52 | import qualified Text.Libyaml as Y
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by DanBurton** [comment]
This is presumably due to today's upgrade on Stackage nightly to `yaml-0.11`.

See: http://hackage.haskell.org/package/yaml-0.11.0.0/changelog

**Comment by lykahb** [comment]
Thank you for bringing this up. I put an upper bound on yaml. Once yaml-0.11 is in stackage LTS (it has 0.8.32 now), I'll add libyaml and bump the lower bound to `yaml >= 0.11`

**Comment by DanBurton** [comment]
> Once yaml-0.11 is in stackage LTS (it has 0.8.32 now), I'll add libyaml and bump the lower bound to yaml >= 0.11

No need to rush to address this, but note that lts-13.0 has been released today with yaml 0.11.
